feature-utils/poly-look/src/react-components/buttons/button.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import "./button.css";
+
+/**
+ * Basic button.
+ * In addition to `label` any valid button HTML attributes are allowed.
+ * @param {Object} props
+ * @param {string} [props.label] - Button text
+ */
+
+const Button = ({ label, ...otherProps }) => {
```

Comment:
I know that this was here before, but it's generally A Good Thing (TM) to be consistent between the component and the CSS class name. In this case I guess you can't use `button` for class, but maybe this could be changed to PolyButton? That way it's more clearly different from the lower-case `button`.

Reply:
The naming for this one did gave me some headaches. Names that came to mind were: `PolyButton` , `PrimaryButton`, `FilledButton`, `BasicButton`  but in the end I've decided to go with the same naming as in the design system. Changing the class name will come with breaking changes in the features and I want to avoid/postpone that for a little longer.  I'm currently more inclined towards `BasicButton` just because we didn't prefix any of our other components.

Reply:
I'm good with any of them, as long as the class is the same.

Reply:
Button has been renamed to `PolyButton`.

Reply:
While I'm OK with this, I think you've got it a bit backwards there: JS symbols can be namespaced, in this case if we have a `Button` that we import from the poly-look package, there's no name clash with anything else as long as we don't import it under the same name. We can even rename or group upon importing, so there we have no need to prefix anything. It _would_ be a little weird to have `button` and `Button`, so I guess I'm OK with prefixing it somehow, I don't think `PolyButton` is ideal, but we can always rename it later.

What we **need** to prefix, however, is CSS, because all the classes exist in the same global namespace. So if we have a `Foobar` component, we need to give it a `poly-foobar` class. So **CSS class and component name should absolutely not be the same** - unless we do prefix a component with `Poly` for some reason, like you did here.

Reply:
As a matter of fact, components and CSS usually have the same name here. The fact that it was not is because button was such a generic name. I don't care about it being called polyButton. I only suggested that name to make it consistent with the CSS class. To be clear, I didn't specifically suggest prefixing, so no need to thumb me down there.

Reply:
I understood is as a suggestion to always have CSS class and component name be the same, e.g. `ImageButton` would have a CSS class `image-button`, or it would be `PolyImageButton` and `poly-image-button`. What I'm saying is: It should be `IconButton` and `poly-icon-button`. Sorry if I misunderstood you there!

Reply:
We're on the same page here, only I understood consistency at a local level (the poly- prefix was only used in button) and you at a global level. If the rule is to *always* add a prefix in CSS, I'm all for that.

Reply:
So, since this has turned into a bit of controversy and with an other component name class mismatch just pushed, I just want to make clear that the end goal of this PR is not to implement all the buttons. As you can see [here](https://www.figma.com/file/qIrr4QJrmYGqVQHQoCECux/polyPod-design-system?node-id=2891%3A2856) there is a lot that is missing and when we add those, I expect the current `polyButton.jsx` to look completely different and maybe under a new name.

Going forward it would be nice if we can settle and start enforcing on how we do this component and css class name, because currently we don't follow the css prefix rule.

Reply:
I'm good with the state it's now. We have kinda settled on the fact that consistency is good, since we have local consistency I'm good with that. Going forward, we could try and enforce some other kind of consistency, but it would involve a lot of tooling so local is good enough.